src/Ledger/Rewards.lagda

```diff
@@ -424,13 +424,101 @@ reward pp blocks rewardPot poolParams stake delegs total = rewards
 
 \subsection{Reward Update}
 \label{sec:reward-update}
-TODO: This section defines the \AgdaRecord{RewardUpdate} type,
+This section defines the \AgdaRecord{RewardUpdate} type,
 which records the net flow of Ada due to paying out rewards
 after an epoch.
-NOTE: The function \AgdaFunction{createRUpd} calculates the
+This type is defined in \Cref{fig:functions:RewardUpdate}.
+The update consists of four net flows:
+\begin{itemize}
+  \item \AgdaField{Δt}: The change to the treasury. This will be a positive value.
+  \item \AgdaField{Δr}: The change to the reserves. This will be a negative value.
+  \item \AgdaField{Δf}: The change to the fee pot. This will be a negative value.
+  \item \AgdaField{rs}: The map of new individual rewards,
+    to be added to the existing rewards.
```

Comment:
If we know for a fact that some of these fields are to be interpreted as positive/negative numbers which not make them correct by construction using naturals?

Reply:
There's a second issue here, which is that the claim for the reserves isn't true: https://github.com/IntersectMBO/cardano-ledger/issues/4771

Reply:
I have weakened the claim to "We typically expect this to be a negative value".